.vale.ini

```diff
@@ -0,0 +1,38 @@
+StylesPath = .vale/styles
+
+MinAlertLevel = suggestion
+
+IgnoredScopes = code, tt, img, url, a, body.id
+
+SkippedScopes = script, style, pre, figure, code, tt, blockquote, listingblock, literalblock
+
+Packages = RedHat, AsciiDoc
+
+Vocab = Foreman
+
+# Use local Vocab terms
+Vale.Terms = YES
+Vale.Avoid = YES
+
+# Ignore files in dirs starting with `.` to avoid raising errors for `.vale/fixtures/*/testinvalid.adoc` files
+[[!.]*.adoc]
+
+BasedOnStyles = RedHat, AsciiDoc
+
+# Match INI files. See: https://docs.errata.ai/vale/scoping
```

Comment:
This URL goes to a domain grabber. Did you mean https://vale.sh/docs/topics/scoping/?